### ISSUE TITLE
Create CMS accessibility audit framework

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9680]
## Background

From the lessons learned through #9679, a framework/path forward for CMS audits will be determined. In order to scale the audit plan, the baseline framework should be crafted and documented.  

## Acceptance Criteria

- [x] POC reviewed with PO for direction

**Comment by EWashb** [comment]
framework determined via #9679 

[WIP doc](https://docs.google.com/spreadsheets/d/1kduP98QBltqtFG8sIB-h-vLXkEoZNPFJ14lEfT6icus/edit#gid=1653727169) will be a living doc
